tuneme/static/css/styles.css

```diff
@@ -75,6 +75,7 @@ input[type='submit']:focus, a.button:focus {opacity:0.5;}
         .grey a.button.inverted {color:#828282; background:white; border-color:#828282;}
 
 
+.decoration {text-decoration: none;}
```

Comment:
@miltontony if I use `a {text-decoration: none;}` then for feature phones all the links will be without underline